src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -176,7 +180,10 @@ private static T ConvertFromInvokeBindingResponse<T>(Autogenerated.InvokeBinding
                 serializedData = TypeConverters.ToAny(data, this.jsonSerializerOptions);
             }
 
-            _ = await this.MakeInvokeRequestAsync(appId, methodName, serializedData, httpExtension, cancellationToken);
+            Autogenerated.InvokeServiceRequest request;
+            CallOptions callOptions;
+            (request, callOptions) = this.MakeInvokeRequestAsync(appId, methodName, serializedData, httpExtension, cancellationToken);
```

Comment:
these can be cleaned up to `var (request, callOptions) = ` there's a few more cases